changelogs/current.yaml

```diff
@@ -71,6 +76,10 @@ bug_fixes:
 - area: router check tool
   change: |
     Fixed a bug where the route coverage is not correctly calculated when a route has weighted clusters.
+- area: zone-aware routing
+  change: |
+    Fixed a bug where zone-aware routing doesn't proportion traffic evenly across upstream hosts
+    if the local and upstream clusters have the same number of zones, but the zone sets aren't identical.
```

Comment:
nit: I would probably move this into the other item, just to make it clear in one place what is changing.